.github/workflows/tarpaulin.yml

```
@@ -2,6 +2,7 @@ name: tarpaulin
 
 on:
   pull_request:
+    paths: ["**/*.rs", "**/*.toml", "**/*.lock", "**/tarpaulin.yml"]
```

```suggestion
    paths:
      - "**/*.lock"
      - "**/*.rs"
      - "**/*.toml"
      - "**/tarpaulin.yml"
```

- For better readability, the patterns should be rather sorted by alphabet.
- In case there should be any patterns to add in the future, the bullet list notation should be preferred as it allows for easier editing.